src/Compiler/Checking/CheckFormatStrings.fs

```diff
@@ -317,13 +317,10 @@ let parseFormatStringInternal
             |> Seq.takeWhile (fun c -> c = '%')
             |> Seq.length
         if delimLen <= 1 && fmt[i..(i+1)] = "%%" then
-            match context with
-            | Some _ ->
-                specifierLocations.Add(
-                    (Range.mkFileIndexRange m.FileIndex
-                        (Position.mkPos fragLine fragCol)
-                        (Position.mkPos fragLine (fragCol+2))), 0)
-            | None -> ()
+            specifierLocations.Add(
```

Comment:
Why are conditions deleted here?

Reply:
Well, long story short, they shouldn't be removed, so good catch and thanks!

Reply:
Also, happy I helped, FYI - this is in two places, one is still there :)